client/src/ui/deckbuilder/DeckBuilderColumn.vue

```diff
@@ -0,0 +1,132 @@
+<template>
+  <div
+      class="column"
+      @dragover="dragOver"
+      @dragleave="dragEnd"
+      @dragend="dragEnd"
+      @drop="drop"
+  >
+    <div
+        v-for="(card, index) in column"
+        class="card"
+    >
+      <img
+          class="card-img"
+          :src="getImageSrc(card)"
+          :alt="card.name"
+          :data-index="index"
```

Comment:
Instead of binding the data here, you can bind it to the event handler, e.g.

```
@dragstart="dragStart($event, index)"
```

(`$event` refers to the original event)